### PR TITLE
new plugin DontRoundMyTimestamps: round 7.6y -> 7y instead of 8y

### PR DESCRIPTION
**Pull request by KontrollFreek**
Stops relative timestamps (<t:1000:**R**>) from using Math.round.

### CODE REVIEW

File: src/plugins/lastfm/index.tsx [hunk 1]
```diff
@@ -101,81 +101,86 @@ function setActivity(activity: Activity | null) {
```
Comment (MrDiamondDog) [inline src/plugins/lastfm/index.tsx:101]: This entire file's changes are completely unrelated and unnecessary

Comment (KontrollFreek) [reply src/plugins/lastfm/index.tsx:101]: i have no clue how PRs work, these are from a previous PR (#2046)

Comment (MrDiamondDog) [reply src/plugins/lastfm/index.tsx]: You have to make different branches on your fork so the changes aren't combined

File: src/plugins/dontRoundMyTimestamps/index.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+*/
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { moment } from "@webpack/common";
+
+export default definePlugin({
+    name: "DontRoundMyTimestamps",
+    authors: [Devs.Lexi],
+    description: "Stops relative timestamps from being rounded and therefore incorrect.",
```
Comment (Vendicated) [inline src/plugins/dontRoundMyTimestamps/index.ts:26]: this description and name are misleading. floor is still a round (down)

Comment (Vendicated) [reply src/plugins/dontRoundMyTimestamps/index.ts:26]: something like
```suggestion
    description: "Always rounds relative timestamps down, so 7.6y becomes 7y instead of 8y",
```

**Comment by Vendicated** [comment]
thanks!!
